test/TestSites/web.config

```diff
@@ -6,6 +6,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath=".\TestSites.exe" arguments="" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="true" />
+    <aspNetCore stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="true" />
```

Comment:
Probably generate less confusion and fewer support issues to have both with no values and a comment  in the file that the values will be provided by the `publish-iis` tool.
```xml
<!-- The values for 'processPath' and 'arguments' will be injected by the 'publish-iis' tool when the 
application is published. -->
<aspNetCore processPath="" arguments="" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" 
forwardWindowsAuthToken="false" />
```